tests/test_date_parser.py

```diff
@@ -245,6 +306,9 @@ def wrapped(date_string):
         self.add_patch(patch('dateparser.date.date_parser', new=self.date_parser))
         self.parser = DateDataParser()
 
+    def given_configuration(self, key, value):
+        settings.update(key, value)
```

Comment:
It is better to patch settings object here. Then we would not rely on `reload_settings` later. Tests have tendency to be copy-pasted and with patch we would have more guaranteed way of rolling back.